### ISSUE TITLE
Gulp watch doesn't ignore child theme js causing infinite scripts loop

### ISSUE THREAD

**Issue by sinthetyc** [understrap/understrap#1307]
## Issue is about..
  - [1] a Bug Report
  - [ ] a Feature Request
  - [ ] a Suggestion
  - [ ] a Question
  - [ ] I need help

## Please tell us about your environment:

  - **Understrap Version:** 5.7.2
  - **Understrap Child Version:** 0.6.0

---

# Issue Overview for Bugreports
Gulp watch is not configured to ignore /js/child-theme.js and /js/child-theme.min.js. This causes an infinite task loop.

## Steps to reproduce?
1. Start gulp watch.
2. Edit /src/js/custom-javascript.js and save.

## Expected behavior
Gulp watch should finish 'scripts' without restarting.

### Other Comments
gulp.watch(
[
	paths.dev + '/js/**/*.js',
	'js/**/*.js',
	'!js/theme.js',
	'!js/theme.min.js',
	'!js/child-theme.js',
	'!js/child-theme.min.js',
],
gulp.series( 'scripts' )
);

**Comment by bacoords** [comment]
This issue was just fixed in a recent PR on the child theme: https://github.com/understrap/understrap-child

Please pull the latest version and reach out if you're still having issues.

**Comment by sinthetyc** [comment]
PR : https://github.com/understrap/understrap-child/pull/284

Looks good. Working as expected.

**Comment by currentcreative** [comment]
Hi, the new version still isn't correct. Note that **line 141** does not have 'paths.dev + ' in front. 

It **should** be this:

gulp.watch(
		[
			paths.dev + '/js/**/*.js',
			**paths.dev + 'js/**/*.js',**
			'!js/child-theme.js',
			'!js/child-theme.min.js',
		],

I figured this out by wondering why Browser Sync wasn't reloading after I added some JavaScript to src/js/custom-javascript.js and then examining your latest gulpfile.js in the understrap-child linked above 3 days ago.
